### DISCUSSION TITLE
Edit a record

### DISCUSSION THREAD

**Discussion by Awesome-Matt** [simonw/datasette#2062] [Q&A]
Sorry to ask this basic question.
Does the web UI of datasette have function of editing a record value?

**Comment by Georgekillstein** [comment]
https://freeaiall.com/ai-news/gpt-5-will-have-a-ph-d-there-is-a-level-of-intelligence-it-is-also-fast/

**Comment by simonw** [comment]
It will in the next release!
- #2781
